assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -175,10 +175,13 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
 
   return (
     <Movable
-      className={`default-movable ${hideHandles ? 'hide-handles' : ''}`}
+      className={`default-movable ${hideHandles ? 'hide-handles' : ''} ${
+        selectedElement.type === 'text' ? 'type-text' : ''
+      }`}
       zIndex={0}
       ref={moveable}
       target={targetEl}
+      edge={true}
```

Comment:
Hmm, would have assumed that the `edge` would also respect the directions defined for the handles.
Perhaps it would be good to ask in the Moveable repo if that's purposefully not so.

Reply:
To be really flexible (and work for our cases) it should accept (also? `true=respect handles def|array=custom`) array of directions, for our 1st use case we need `top edge` but not `top handle` and 2nd: `vertical edges only`.
Not sure if they would like to add it because it can be solved with CSS.